### DISCUSSION TITLE
Calendar view related-function

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#9708] [Q&A]
Does the calendar view support the following features?

Assuming that the meeting starts at 7am and lasts for 2 hours, then 7-9am should be displayed in the calendar as a meeting. Now the default is to only show meetings from 7-8am.

However, the current calendar view settings seem to only organize one field. Is there a way to do the above?

**Comment by dstala** [comment]
Not supported currently, but is in our roadmap. You can upvote this feature request here
https://github.com/nocodb/nocodb/issues/8583
